test/highlight/basics.scala

```diff
@@ -0,0 +1,37 @@
+object Hello {
```

Comment:
Tree sitter has builtin support for unit testing highlight rules: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#unit-testing